### PR TITLE
Fixed project name on scripts

### PR DESCRIPTION
**Pull request by avelino**

### CODE REVIEW

File: scripts/windows/install-as-service.bat [hunk 1]
```diff
@@ -6,19 +6,19 @@
 :: https://nssm.cc/download
 :: This script itself should run in the gogs path, too.
 :: In case of startup failure, please read carefully the log file.
-:: Make sure Gogs work running manually with "gogs web" before running
+:: Make sure Gitea work running manually with "gogs web" before running
```
Comment (lunny) [inline scripts/windows/install-as-service.bat]: gitea web

**Comment by lunny** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

File: scripts/init/centos/gitea [hunk 2]
```diff
@@ -27,7 +27,7 @@ NAME=gitea
 GITEA_HOME=/home/git/gitea
 GITEA_PATH=${GITEA_HOME}/$NAME
 GITEA_USER=git
-SERVICENAME="Gogs Go Git Service"
+SERVICENAME="Gitea Git with a cup of tea"
```
Comment (bkcsoft) [inline scripts/init/centos/gitea]: Maybe add a dash between Gitea and Git, like so `Gitea - Git with a cup of tea`
Not sure what the guidelines for CentOS initrc is though.

File: scripts/init/suse/gitea [hunk 3]
```diff
@@ -21,7 +21,7 @@ NAME=gitea
 GITEA_HOME=/home/git/gitea
 GITEA_PATH=${GITEA_HOME}/$NAME
 GITEA_USER=git
-SERVICENAME="Go Git Service"
+SERVICENAME="Git with a cup of tea"
```
Comment (bkcsoft) [inline scripts/init/suse/gitea]: Forgot `Gitea`, and same as above.

**Review by bkcsoft — CHANGES_REQUESTED** [review]
Maybe we should unify them all into `Gitea - Git with a cup of Tea`? :)

**Comment by avelino** [comment]
@bkcsoft update

**Comment by tboerger** [comment]
Thanks

**Pull request merged**